src/gui/uithememanager.cpp

```diff
@@ -259,6 +273,23 @@ QIcon UIThemeManager::getSystrayIcon() const
 
 Path UIThemeManager::getIconPath(const QString &iconId) const
 {
+#if (defined(Q_OS_UNIX) && !defined(Q_OS_MACOS))
+    if (m_useSystemIcons)
+    {
+        Path path = Utils::Fs::tempPath() / Path(iconId + u".png");
```

Comment:
What is happening here? Why are you looking in `TEMP`?

Reply:
This is legacy code. IIRC, previously it helped to provide icons from the system theme to the WebUI, for which we had to save them to a temporary folder. Now it is still used when you need to get a scaled pixmap from some icon, but this can be avoided, so this code can be discarded as soon as #18269 is merged.